### ISSUE TITLE
get tag=103 err=50 when use ldap-ltb to change password

### ISSUE THREAD

**Issue by lordk911** [jp-gouin/helm-openldap#156]
**Describe the bug**
get tag=103 err=50 when use ldap-ltb to change password
but I can use phpldapadmin to change password

**To Reproduce**
I install helm-openldap use master branch : 
```
helm list -n openldap
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
openldap        openldap        1               2024-03-20 15:20:47.555650925 +0800 CST deployed        openldap-stack-ha-4.2.2 2.6.6    
```
Try to change user's password by ldap-ltb

**Screenshots**
The log when use phpldapadmin to change password : 
![image](https://github.com/jp-gouin/helm-openldap/assets/19989300/786851df-3cd3-4ef4-892d-8b7fc4a55516)

The log when useldap-ltb to change password : 
![image](https://github.com/jp-gouin/helm-openldap/assets/19989300/cd49a8fe-dea7-4928-8e24-b2affc1eb063)

**Additional context**

**Comment by lordk911** [comment]
https://github.com/ltb-project/self-service-password/issues/800#issuecomment-1711274009

**Comment by jp-gouin** [comment]
Fix in latest version `v4.2.4` via PR #165
